.github/release-drafter.yml

```diff
@@ -0,0 +1,36 @@
+name-template: 'v$RESOLVED_VERSION'
```

Comment:
What's this file do, Alex? I'm asking since I've never seen it before.

Reply:
see the [release-drafter github Action](https://github.com/release-drafter/release-drafter).  My understanding of this is that it's used to configure the release notes, which seemed like something that would be useful.

I've based the majority of the github build configuration on what Brandon has done [here](https://github.com/caesarsdigital/zio-diffx) and Calvin has done [here](https://github.com/kaizen-solutions/virgil).  I admit to not working with Actions in the past, so I'm riding the coattails of others.